### PR TITLE
docs: Update that the Tour De Sun 22 is now closed

### PR DESCRIPTION
**Pull request by web3validator**
#### Problem
The current documentation includes outdated information about the Tour De Sun 22 program, which is now closed to new applicants. This could cause confusion for users who are trying to participate in the program.

#### Summary of Changes
- Updated the Tour De Sun 22 section to indicate that the program is winding down and applications are now closed.
- Added information that March 2024 was the final month of the program.
- Included a link to the official blog post announcing the end of Tour De Sun 22 for further details.

Fixes #

### CODE REVIEW

File: docs/src/operations/validator-initiatives.md [hunk 1]
```diff
@@ -13,12 +13,15 @@ The Solana Foundation helps facilitate the growth of the consensus validator net
...
 
 ## Solana Foundation Server Program
 
 Separately from the delegation program, The Solana Foundation offers a server program that provides servers in various data-centers all over the world. If you would like to run a consensus validator or RPC node, you may use this program to rent bare metal servers in various data-centers. The servers meet or exceed the [Solana validator hardware specs](./requirements.md#hardware-recommendations). No long-term lease commitments are required. To find out more, visit the [Solana server program page](https://solana.org/server-program).
 
 ## Stake Pools
 
-The Solana ecosystem operates various stake pools.  These stake pools delegate stake to validators using various delegation strategies. To learn more about stake pools and their delegation strategies, visit the [Solana Foundation stake pools page](https://solana.org/stake-pools).
\ No newline at end of file
+The Solana ecosystem operates various stake pools.  These stake pools delegate stake to validators using various delegation strategies. To learn more about stake pools and their delegation strategies, visit the [Solana Foundation stake pools page](https://solana.org/stake-pools).
```
Comment (steviez) [inline docs/src/operations/validator-initiatives.md]: This chunk shouldn't be in the diff - I don't see any changes to the actual content so perhaps a line ending issue ?

File: docs/src/operations/validator-initiatives.md [hunk 2]
```diff
@@ -13,12 +13,15 @@ The Solana Foundation helps facilitate the growth of the consensus validator net
 
 ## Tour De Sun 22
 
-Delegation program participants who operate a performant testnet node, may also join Tour De Sun 22 (TdS22).  The program gives locked SOL incentive payments to operators for running approved hardware on Testnet. For more information, see [TdS22](https://solana.org/tds22).
+Tour De Sun 22 is winding down. Applications to the program are now closed.
+March 2024 was the final month of the program. The program is closed and will not accept new applicants.
+
+Please see the [End of Tour De Sun 22 blog post](https://solana.com/news/solana-foundation-announces-end-of-tour-de-sun-22) for more information.
```
Comment (steviez) [inline docs/src/operations/validator-initiatives.md]: @tigarcia - Any thoughts on this? I don't necessarily like duplicating the same content from this page:
https://solana.org/tds22

But, I also see the point about not having outdated information. Some options come to mind:
- Remove this section from this doc altogether
- Put a very brief note here to serve as a tombstone, something like below

> The Tour de Sun 22 (TdS22) has wound down, see [TdS22](https://solana.org/tds22) for more information

Comment (web3validator) [reply docs/src/operations/validator-initiatives.md]: Put a very brief note here to serve as a tombstone

File: docs/src/operations/validator-initiatives.md (hunk 1 above)
Comment (steviez) [reply docs/src/operations/validator-initiatives.md]: This item is not resolved - there are no changes to this text so this line should not be in the diff

File: docs/src/operations/validator-initiatives.md (hunk 2 above)
Comment (steviez) [reply docs/src/operations/validator-initiatives.md]: I think we should go even briefer:

> The Tour De Sun 22 is closed and no longer accepting applications. Please see the [End of Tour De Sun 22 blog post](https://solana.com/news/solana-foundation-announces-end-of-tour-de-sun-22) for more information.

**Comment by steviez** [comment]
Just incase - I see you merged a commit, but I don't see any actual differences / the commit you pushed is empty. And by the way, it could be a line ending issue for why the extra lines are showing up at the end of the diff. Are you editing on a Windows machine ?

**Comment by web3validator** [comment]
I have edited the PR. Kindly check if it looks correct now.

File: docs/src/operations/validator-initiatives.md [hunk 3]
```diff
@@ -13,12 +13,12 @@ The Solana Foundation helps facilitate the growth of the consensus validator net
 
 ## Tour De Sun 22
 
-Delegation program participants who operate a performant testnet node, may also join Tour De Sun 22 (TdS22).  The program gives locked SOL incentive payments to operators for running approved hardware on Testnet. For more information, see [TdS22](https://solana.org/tds22).
+The Tour De Sun 22 is closed and no longer accepting applications. Please see the [End of Tour De Sun 22 blog post](https://solana.com/news/solana-foundation-announces-end-of-tour-de-sun-22) for more information.
```
Comment (steviez) [inline docs/src/operations/validator-initiatives.md]: Oops, my mistake; forgot a word here
```suggestion
The Tour De Sun 22 program is closed and no longer accepting applications. Please see the [End of Tour De Sun 22 blog post](https://solana.com/news/solana-foundation-announces-end-of-tour-de-sun-22) for more information.
```

**Review by steviez — COMMENTED** [review]
> I have edited the PR. Kindly check if it looks correct now.

Thanks for the continued additions. The first segment needs one more word (this was a mistake on my part, apologies). However, the last line of the file is still showing as changed but should NOT be:
image

As I mentioned, this might be an issue with your editor / system using different line endings

File: docs/src/operations/validator-initiatives.md (hunk 1 above)
Comment (web3validator) [reply docs/src/operations/validator-initiatives.md]: Yes, it is the problem with editor

Comment (web3validator) [reply docs/src/operations/validator-initiatives.md]: I fixed it, @steviez  check it.

**Review by steviez — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**